pkgs/development/python-modules/cffi/default.nix

```diff
@@ -33,12 +33,32 @@ if isPyPy then null else buildPythonPackage rec {
     "-Wno-unused-command-line-argument -Wno-unreachable-code";
 
   doCheck = !stdenv.hostPlatform.isMusl && !stdenv.isDarwin; # TODO: Investigate
-  checkPhase = ''
-    py.test -k "not test_char_pointer_conversion"
-  '';
+  disabledTests = [
+    # unix gets mistaken as windows because find_library returns None
```

Comment:
The test diagnostics could be better, but the fact that `find_library('c')` returns None and the test falls through to the Windows case indicates that find_library is broken.